src/reporter.test.ts

```diff
@@ -375,7 +375,7 @@ describe('reporter', () => {
               },
               response: {
                 truncated: false,
-                body: `{"key":"****","password":"****","e":"${value}","secret":"****","f":"${value}","g":"${value}","h":"${shorter}"}...[too long]`,
+                body: `{"key":"****","password":"****","e":"${value}","secret":"****","f":"${value}","g":"${value}","h":"${shorter}✂"}`,
```

Comment:
regardless of what truncation indicator we go with, i would suggest using a constant to avoid brittleness

Reply:
Actually, I am very much against using constants in tests: you DO want to know if something breaks the contract with the backend. That character is the contract. If you can change it with just one keystroke, it's way too easy to break the contract.